packages/types/src/interfaces/contractsAbi/definitions.ts

```diff
@@ -0,0 +1,173 @@
+// Copyright 2017-2020 @polkadot/types authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// const layout = {
+//   InkLayoutArray: {
+//     cells_per_elem: 'u32',
```

Comment:
Always camelCase

Reply:
This is directly based on the ink! 3.0 abi format, designed to be created from json...not sure how to use camelcase instead, though it's not necessary for now

Reply:
Ummm... then that is wrong/weird there. Seems there are some camelCase in that JSON and some not. Having said that, I believe the actual JSON paring does take care of anything non-camel, well, it _used_ to, will need a check. (If it doesn't, can't leave it as-is, but then will apply a fix there)

Reply:
Yeap, here, it attempts to... https://github.com/polkadot-js/api/blob/master/packages/types/src/codec/Struct.ts#L17-L20

Reply:
Ok, well, it _used_ to, doesn't seem to work. Need to apply a fix there.

Reply:
https://github.com/paritytech/ink/pull/514